com.github.emilianavt.OpenSeeFace.yml

```diff
@@ -0,0 +1,22 @@
+id: com.github.emilianavt.OpenSeeFace
+branch: master
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: 'facetracker.sh'
+finish-args:
+    - "--device=all"
+    - "--share=network"
```

Comment:
no display permissions?

does it really need network access?

Reply:
This app is meant to be used as a CLI app, so no display permissions needed.
It also requires network access to output to a port on the network for the face tracking data.